jwst/flatfield/flat_field_step.py

```diff
@@ -1,22 +1,36 @@
-#! /usr/bin/env python
-
 from ..stpipe import Step
 from .. import datamodels
 from . import flat_field
 
 # For the following types of data, it is OK -- and in some cases
 # required -- for the extract_2d step to have been run.  For all
 # other types of data, the extract_2d step must not have been run.
-EXTRACT_2D_IS_OK = ["NRS_LAMP", "NRS_BRIGHTOBJ", "NRS_FIXEDSLIT",
-                    "NRS_MSASPEC"]
-
-# NIRSpec imaging types (see exp_type2transform in assign_wcs/nirspec.py).
-NRS_IMAGING_MODES = ["NRS_IMAGE", "NRS_FOCUS",
-                     "NRS_TACQ", "NRS_MSATA", "NRS_TASLIT",
-                     "NRS_CONFIRM", "NRS_TACONFIRM",
-                     "NRS_MIMF", "NRS_WATA"]
-# Supported NIRSpec spectrographic types.
-NRS_SPEC_MODES = ["NRS_BRIGHTOBJ", "NRS_FIXEDSLIT", "NRS_MSASPEC", "NRS_IFU"]
+EXTRACT_2D_IS_OK = [
+    "NRS_BRIGHTOBJ",
+    "NRS_FIXEDSLIT",
+    "NRS_LAMP",
+    "NRS_MSASPEC",
+    ]
+
+# NIRSpec imaging types (see exp_type2transform in assign_wcs/nirspec.py)
+NRS_IMAGING_MODES = [
+    "NRS_CONFIRM",
+    "NRS_FOCUS",
+    "NRS_IMAGE",
+    "NRS_MIMF",
+    "NRS_MSATA",
+    "NRS_TACONFIRM",
+    "NRS_TACQ",
+    "NRS_TASLIT",
+    "NRS_WATA",
+    ]
+# Supported NIRSpec spectrographic types. No flat fielding for NRS_AUTOFLAT
+NRS_SPEC_MODES = [
+    "NRS_BRIGHTOBJ",
+    "NRS_FIXEDSLIT",
+    "NRS_IFU",
+    "NRS_MSASPEC",
+    ]
```

Comment:
Is this formatting done by your IDE? I find vertical harder to read than horizontal but tha's probably just me.

Reply:
😂

We should start using [black](https://github.com/python/black) and then we would never have to worry about this again.  =)